add some info (#1)
ia-toki/apio-2020-web: contestants.md

---
layout: single-page-md
title: APIO 2020 Contestants
key: contestants
---

> The country listed below is the country that has been participated in APIO in the past.

Reviewer: ```suggestion
The countries listed below are the countries that have participated in APIO in the past.
```

> If you think that your country is eligible but has not been listed here, please contact us at [info@toki.id](mailto:info@toki.id).

Reviewer: ```suggestion
If you think that your country is eligible but has not been listed here, please contact us at [info.apio2020@toki.id](mailto:info.apio2020@toki.id).
```
